gcc/jit/jit-playback.cc

```diff
@@ -675,6 +675,9 @@ new_function (location *loc,
     /* See handle_pure_attribute in gcc/c-family/c-attribs.cc. */
     else if (attr == GCC_JIT_FN_ATTRIBUTE_PURE)
       DECL_PURE_P (fndecl) = 1;
+    /* See handle_pure_attribute in gcc/c-family/c-attribs.cc. */
```

Comment:
Type: you wrote `pure` instead of whatever the function is called.

Reply:
Woups.